frontend/src/metabase/parameters/components/ParameterValueWidget.jsx

```diff
@@ -253,7 +252,8 @@ function getFields(metadata, parameter) {
 
 function getFieldIds(parameter) {
   const { field_ids = [], field_id } = parameter;
-  return field_id ? [field_id] : field_ids;
+  const fieldIds = field_id ? [field_id] : field_ids;
+  return fieldIds.filter(id => typeof id === "number");
```

Comment:
Only allow `id` of type `number` as it is the only type that the API will accept.